test/issue103-single-file-package.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/sh
 set -e
 cd ${CURR_DIR}
+rm -f single-file-test
 ./issue103-single-file-package.d foo -- bar
+if [ -f single-file-test ]; then
+	echo "Shebang invocation produced binary in current directory"
+	exit 1
+fi
+
 ${DUB} run --single issue103-single-file-package-json.d --compiler=${DC}
-${DUB} issue103-single-file-package-w-dep.d --compiler=${DC}
+if [ ! -f single-file-test ]; then
+	echo "Normal invocation did not produce a binard in the current directory"
+	exit 1
+fi
+rm single-file-test
+
+${DUB} issue103-single-file-package-w-dep.d
```

Comment:
Since this is using the "shebang" short syntax, the test would basically be redundant with the first one. Do you see value in it anyway?